desktop/server.js

```diff
@@ -21,10 +21,12 @@ if (NO_SERVER) {
     if (jsonStats.errors.length > 0) {
       throw jsonStats.errors.join('\n')
     }
+
+    console.log(stats)
+
     if (jsonStats.warnings.length > 0) {
```

Comment:
the unused files are a warning so lets just let that through and output it. don't die